lingetic-nextjs-frontend/app/page.tsx

```
@@ -1,10 +1,317 @@
+function CTASection() {
+  return (
+    <section className="py-20 px-4 md:px-8 lg:px-24 bg-gradient-to-r from-blue-600 to-blue-700">
+      <div className="max-w-4xl mx-auto text-center">
+        <h2 className="text-3xl md:text-4xl font-bold mb-6 text-white">
+          Ready to Start Your Language Journey?
+        </h2>
+        <p className="text-xl mb-10 text-white opacity-90">
+          Join Lingetic today and start having real conversations in your target
+          language.
+        </p>
+        <button className="bg-white text-[#2563eb] hover:bg-blue-50 px-8 py-6 text-lg">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)